sentinel/src/transitions/accept_patient_reports.js

```diff
@@ -175,6 +213,7 @@ const handleReport = (doc, config, callback) => {
 
       addMessagesToDoc(doc, config, registrations);
       addRegistrationToDoc(doc, registrations);
+      addReportUUIDToRegistration(doc, registrations, callback);
```

Comment:
This is a good start, but is not enough to have this work as needed: your registration is still not blocking later lines of code from executing as it should.

It's definitely worth reading about and going through some tutorials on callbacks and promises to make sure you are familiar with how Javascript works with these tools. It's reasonably complicated when you first encounter it (and sometimes after!) and it's critical to how Javascript works.

Let's also jump on a Slack call when you're around and we can run through some examples together :-)